src/connect/components/form-input/form-input.stories.tsx

```diff
@@ -79,7 +79,7 @@ export const WithCustomErrorMessage: Story = {
         required: true,
         customErrorMessages: {
             valueMissing: (
-                <span className="text-pink-500">This field is required</span>
```

Comment:
this color style no longer exists in our tailwind config after adding our style guide colors. the plugin caught it